hack/lib/util.sh

```diff
@@ -139,6 +135,23 @@ kube::util::gen-doc() {
   ${cmd} "${tmpdir}"
   # create the list of generated files
   ls "${tmpdir}" | LC_ALL=C sort > "${tmpdir}/.files_generated"
+
+  # remove all old generated file from the destination
+  for file in $(cat "${dest}/.files_generated" 2>/dev/null); do
```

Comment:
This is usually bad form (`for i in $(cat ...)`), though I can see it is pre-existing here.

If what you have is one file name per line, then consider the superior:

```shell
  while read file; do
    ...
  done <"${dest}/.files_generated"
```